src/System.Net.Requests/src/System.Net.Requests.csproj

```diff
@@ -52,6 +52,10 @@
     <Compile Include="$(CommonPath)\System\NotImplemented.cs">
       <Link>Common\System\NotImplemented.cs</Link>
     </Compile>
+    <ProjectReference Include="..\..\System.Net.Http\src\System.Net.Http.csproj">
```

Comment:
Discussed offline with @ericstj , it is not required to service every assembly depending on another serviced assembly.